scripts/codemod/cli.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * inspire by react-code-mod CLI: https://github.com/reactjs/react-codemod/blob/master/bin/cli.js
+ */
+
+// hsds-codemod optional-name-of-transform path/to/src [...options]
+
+const globby = require('globby')
+const inquirer = require('inquirer')
+const meow = require('meow')
+const path = require('path')
+const execa = require('execa')
+const chalk = require('chalk')
+const isGitClean = require('is-git-clean')
+
+const transformerDirectory = path.join(__dirname, 'transforms')
+
+const jscodeshiftExecutable = require.resolve(
+  '../../node_modules/jscodeshift/bin/jscodeshift.sh'
+)
+
+function checkGitStatus(force) {
+  let clean = false
+  let errorMessage = 'Unable to determine if git directory is clean'
+  try {
+    clean = isGitClean.sync(process.cwd())
+    errorMessage = 'Git directory is not clean'
+  } catch (err) {
+    if (err && err.stderr && err.stderr.indexOf('Not a git repository') >= 0) {
+      clean = true
+    }
+  }
+
+  if (!clean) {
+    if (force) {
+      console.log(`WARNING: ${errorMessage}. Forcibly continuing.`)
+    } else {
+      console.log('Thank you for using hsds-react-codemods!')
+      console.log(
+        chalk.yellow(
+          '\nBut before we continue, please stash or commit your git changes.'
+        )
+      )
+      console.log(
+        '\nYou may use the --force flag to override this safety check.'
+      )
+      process.exit(1)
+    }
+  }
+}
+
+function runTransform({ files, flags, parser, transformer }) {
+  const transformerPath = path.join(transformerDirectory, `${transformer}.js`)
+
+  let args = []
+
+  const { dry, print } = flags
```

Comment:
+1 love doing a dry run first!

Reply:
verbose would need to be set to `2` to make it valuable. I guess I could rewrote the script to force the verbose value when a dry run is active